### PR TITLE
KNO-255

### PR DESCRIPTION
**Pull request by BrendanQuinnJ1**
https://jupiterone.atlassian.net/browse/KNO-255

### Alerts Rule Packs
- [x] Does a related alert already exist, and should it be tweaked or added to instead?
- [x] Test each query to make sure it works
- [x] Look for hardcoded variables/parameter values in the query
- [x] Consider Severity for Alerts
- [x] Spellcheck
- [x] Use all caps for J1QL keywords and relationship classes
- [x] Upload the alerts rule pack JSON into JupiterOne to validate

### CODE REVIEW

**Review by a-u-h-g — APPROVED** [review]

File: rule-packs/common-alerts.json [hunk 1]
```diff
@@ -72,5 +72,17 @@
...
     ],
     "alertLevel": "MEDIUM"
+  },
+  {
+    "name": "J1-Integration-Failed-Accuracy-1Day",
+    "description": "Find jupiterone_integrations that have failed. This data will be updated every 24 hours and will show the status of the last run of the integration at that point in time",
+    "queries": [
+      {
+        "name": "query0",
+        "query": "Find jupiterone_integration WITH status = 'FAILED',
```
Comment (a-u-h-g) [inline rule-packs/common-alerts.json:82]:
```suggestion
        "query": "FIND jupiterone_integration WITH status = 'FAILED',
```

**Review by a-u-h-g — APPROVED** [review]

**Comment by FrancesCoronel** [comment]
Is it okay to close this PR?

**Comment by a-u-h-g** [comment]
This a good rule, would like this merged and pushed to rule pack in product

**Comment by SeaBlooms** [comment]
this was covered in another PR creating an 'Integrations Monitoring' rule pack

**Pull request closed without merging**